src/Game.java

```diff
@@ -135,27 +127,20 @@ public void checkUserGuess() {
         while (!correctPlayer) {
             userGuess = scanner.nextLine();
 
-            // checks for player in player pool
-            // don't understand code, just knows what it does
-            if (Arrays.stream(playerArrayToString).noneMatch(userGuess::equalsIgnoreCase)) {
+            userPickedPlayer = null;
+            for (Player currentPlayer : playerPool)
+                if (currentPlayer.getName().equalsIgnoreCase(userGuess)) {
+                    userPickedPlayer = currentPlayer;
+                    break;
+                }
+
+            if (userPickedPlayer == null) {
                 System.out.println("Please select a player from the player pool.");
-            } else {
-                // only counts valid guesses
-                count++;
+                continue;
```

Comment:
This keyword means to restart the nearest loop.

See documentation on [Javatpoint](https://www.javatpoint.com/java-continue).